Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/37Y6-create-raidz1-pool/create-raidz1-pool

```diff
@@ -0,0 +1,92 @@
+
+}
+
+node() {
+
+job_id=$(echo $1)
+pipeline_id=$(echo $2)
+case_id=37Y6
+commit_id=$(echo $3)
+source ~/.profile
+gittoken=$(echo "$github_token")
+
+time="date"
+current_time=$(eval $time)
+
+#pooling over previous job to complete
+echo "*******Creating cstor-pool**************"
+bash Openshift-EE/utils/pooling jobname:s2-j3-cstor-mirrored-pool
+bash Openshift-EE/utils/e2e-cr jobname:s2-j4-cstor-raidz1-pool jobphase:Running init_time:"$current_time"
+
+run_id="raidz1";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=cstor-block-device-pool-provision metadata=${run_id})
+echo $test_name
+cd litmus
+
+# copy the content of deployer run_litmus_test.yml into a different file to update the test specific parameters.
+cp providers/cstor/cstor-block-device-pool/run_litmus_test.yml create_raidz1_pool.yml
+
+#################
+## RUNNER MAIN ##
+#################
+echo "Running the litmus test.."
+
+: << EOF
+  ---------------------------------------------------------------------------------------------------------------------------------------------     
+ | specAttribute     | kind   |         default value                           | test specifc value                                           |
+  ---------------------------------------------------------------------------------------------------------------------------------------------
+ | Litmus job label  | label  | app: cstor-block-device-pool-provision          | app: cstor-raidz1-pool-provision                             |
+ | Litmus Job name   | name   | generateName: cstor-block-device-pool-provision | generateName: cstor-block-device-raidz1-pool-provision       |
+ | ImagePullPolicy   | value  | Always                                          | IfNotPresent                                                 |
+ | Pool name         | name   | cstor-block-disk-pool                           | cstor-block-disk-pool-raidz1                                 |
+ | Pool type         | value  | striped                                         | raidz                                                        |
+ | storage class     | env    | openebs-cstor-disk                              | openebs-cstor-raidz1                                         |
+   ---------------------------------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/app: cstor-block-device-pool-provision/app: cstor-raidz1-pool-provision/g' \
+-e 's/generateName: cstor-block-device-pool-provision/generateName: cstor-block-device-raidz1-pool-provision/g' \
+-e 's/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g' \
+-e 's/value: cstor-block-disk-pool/value: cstor-block-disk-pool-raidz1' \
+-e 's/value: striped/value: raidz' \
+-e 's/value: openebs-cstor-disk/value: openebs-cstor-raidz1' create_raidz1_pool.yml
+
```

Comment:
@sushma1118 where is the sed command to append the run_id.

Reply:
OK. Done